app/models/mapping.rb

```diff
@@ -41,10 +41,11 @@ def after_created
         topic2: mappable.topic2.filtered,
         mapping_id: id
       )
-      Events::SynapseAddedToMap.publish!(mappable, map, user, nil)
+      meta = { 'mapping_id': id }
+      Events::SynapseAddedToMap.publish!(mappable, map, user, meta)
```

Comment:
Yeah, so to speak. I hadn't thought of any reason prior to this why it would have been necessary